presto-main/src/main/java/com/facebook/presto/sql/analyzer/FeaturesConfig.java

```diff
@@ -60,7 +60,7 @@
     private boolean legacyOrderBy;
     private boolean legacyMapSubscript;
     private boolean optimizeMixedDistinctAggregations;
-    private boolean distributedSort = false;
+    private boolean distributedSort = true;
```

Comment:
I'm going to enable it in `DistributedQueryRunner`, we don't want to run all of ATQ for distributed_sort=true and distributed_sort=false.